core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -175,6 +244,8 @@ package object config {
     ConfigBuilder("spark.scheduler.listenerbus.metrics.maxListenerClassesTimed")
       .internal()
       .intConf
+      .checkValue(_ > 0,
+        "The maxListenerClassesTimed of listener bus event queue must not be negative")
```

Comment:
Since `_ > 0`, we should avoid 0, too.
`must not be negative` => `must be positive`.